rainbow-me/rainbow
# Settings Redesign

Fixes TEAM2-247, TEAM2-78, TEAM2-77, TEAM2-79

Figma link (if any):
https://www.figma.com/file/Xc4Pq3CLJNVG52z1K0URvP/notifications?node-id=647:4924

## What changed (plus any additional context for devs)
- settings redesign on all settings pages
- typescript everything
- foundation for notification settings
- replaced all existing components with design system equivalents, including screens that weren't explicitly redesigned

## Screen recordings / screenshots
https://www.loom.com/share/ec325c353a5743ce9f00ecc06d7b6879


## What to test
- make sure all settings screens are still there
- make sure no functionality has changed
- make sure the implementation matches the figma designs (see above)
- go to `src/components/settings-menu/SettingsSection.tsx` and enable + test all the settings that may be hidden for various reasons. should be as easy as replacing something like `{conditional + <Component />}` with `{true + <Component />}`


## Final checklist
